File: Biblioteca/src/servicios/MenuImplementacion.java
package servicios;

import java.util.Scanner;

public class MenuImplementacion implements MenuInterfaz{

	Scanner sc = new Scanner(System.in);
	public int menuBiblioteca() {
		
		 System.out.println("Bibliotecas");
		 System.out.println("///////////");
		 System.out.println("[0] - Cerrar Menú");
		 System.out.println("[1] - Crear una nueva Biblioteca");
		 System.out.println("[2] - Entrar en una biblioteca específica");
		 System.out.println("///////////");
		 int op = sc.nextInt();

		 return op;
	}
	
	public int menuParaBibliotecas() {
		
		System.out.println("Opciones de la biblioteca");
		System.out.println("///////////");
		System.out.println("[0] - Cerrar Menú");
		System.out.println("[1] - Dar de alta un nuevo cliente");
		System.out.println("[2] - Dar de alta un nuevo libro");
		System.out.println("[3] - Dar de alta un nuevo prestamo");
		System.out.println("[4] - Estado actual de libros");
		System.out.println("[5] - Libros de la biblioteca");
		System.out.println("///////////");
		int op = sc.nextInt();

		 return op;
	}
}

File: Ejercicio1Global/servicios/MenuImplementacion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ejercicio1Global.servicios
{
    internal class MenuImplementacion : MenuInterfaz
    {
        public int menuBiblioteca()
        {
            Console.WriteLine("Bibliotecas");
            Console.WriteLine("///////////");
            Console.WriteLine("[0] - Cerrar Menú");
            Console.WriteLine("[1] - Crear una nueva Biblioteca");
            Console.WriteLine("[2] - Entrar en una biblioteca específica");
            Console.WriteLine("///////////");
            int op = Convert.ToInt32(Console.ReadLine());

            return op;

        }

        public int menuParaBibliotecas()
        {
            Console.WriteLine("Opciones de la biblioteca");
            Console.WriteLine("///////////");
            Console.WriteLine("[0] - Cerrar Menú");
            Console.WriteLine("[1] - Dar de alta un nuevo cliente");
            Console.WriteLine("[2] - Dar de alta un nuevo préstamo");
            Console.WriteLine("[3] - Dar de alta un nuevo libro");
            Console.WriteLine("[4] - Entrega de Libro");
            Console.WriteLine("[5] - Libros de la biblioteca");
            Console.WriteLine("///////////");
            int op = Convert.ToInt32(Console.ReadLine());

            return op;

        }
    }
}
